index.js

```diff
@@ -14,6 +14,7 @@ const D2LHypermediaRels = {
 	firstName: 'https://api.brightspace.com/rels/first-name',
 	lastName: 'https://api.brightspace.com/rels/last-name',
 	myEnrollments: 'https://api.brightspace.com/rels/my-enrollments',
```

Comment:
aw jeez there's like, a bunch of `my*` things here... I see your point :(

Reply:
Yeah, I can flip or flop either way on this one.